keyboards/clickety_split/leeloo/info.json

```diff
@@ -1,13 +1,32 @@
 {
-  "keyboard_name": "Leeloo",
   "manufacturer": "Clickety Split Ltd",
   "url": "https://clicketysplit.ca/pages/leeloo",
   "maintainer": "ClicketySplit",
   "usb": {
     "vid": "0x4013",
-    "pid": "0x2022",
-    "device_version": "0.1.3"
+    "pid": "0x2022"
   },
+  "diode_direction": "COL2ROW",
+  "matrix_pins": {
+      "cols": ["F6", "F7", "B1", "B3", "B2", "B6"],
+      "rows": ["C6", "D7", "E6", "B4", "B5"]
+  },
+  "encoder": {
+    "rotary": [
+      {"pin_a": "F5", "pin_b": "F4"}
+    ]
+  },
+  "split": {
+      "soft_serial_pin": "D2",
+      "encoder": {
+          "right": {
+              "rotary": [
+                  {"pin_a": "F4", "pin_b": "F5"}
+              ]
+          }
+      }
+  },
+  "debounce": 5,
```

Comment:
```suggestion
```

Reply:
Thank you for the recommendation, @fauxpark!